cgo/errors.go

```diff
@@ -0,0 +1,8 @@
+package cgo
+
+const (
+	ErrInvalidHandle = -1 - iota
```

Comment:
does this  mean the next one is `0` or `-2`?

Reply:
I would suggest just defining them as their cgo versions, this way these are guranteed to match

Reply:
This is -1 as iota starts at 0.